src/Services/LDAPService.php

```diff
@@ -975,7 +975,7 @@ public function updateLDAPFromMember(Member $member)
             foreach ($member->config()->ldap_field_mappings as $attribute => $field) {
                 $relationClass = $member->getRelationClass($field);
                 if ($relationClass) {
-                    // todo no support for writing back relations yet.
+                    continue;
                 } else {
                     $attributes[$attribute] = $member->$field;
                 }
```

Comment:
Why are we adding a `continue;` here? It seems like we should just change the condition like so

```suggestion
                if (!$relationClass) {
                    $attributes[$attribute] = $member->$field;
                }
```

Reply:
Updated